Sources/ArcGISToolkit/Common/AttachmentsFeatureElementView.swift

```diff
@@ -192,10 +191,10 @@ struct AttachmentsFeatureElementView: View {
     func onDelete(attachmentModel: AttachmentModel) async throws -> Void {
```

Comment:
This no longer needs to be `async throws`.
```suggestion
    func onDelete(attachmentModel: AttachmentModel) -> Void {
```